sensor_msgs/PointCloud2[] clouds
geometry_msgs/PoseWithCovarianceStamped[] poses
